Plugin/Magento/Sales/Api/Data/OrderExtension.php

```diff
@@ -10,39 +10,51 @@
 
 use Magento\Framework\App\ObjectManager;
 
-class OrderExtension {
+class OrderExtension
+{
 
+    /**
+     * @var \Magento\Framework\HTTP\PhpEnvironment\Request
+     */
     protected $request;
 
-    public function __construct(\Magento\Framework\HTTP\PhpEnvironment\Request $request) {
+    /**
+     * @var ObjectManager
+     */
+    private $objectManager;
+
+    /**
+     * OrderExtension constructor.
+     *
+     * @param \Magento\Framework\HTTP\PhpEnvironment\Request $request
+     */
+    public function __construct(\Magento\Framework\HTTP\PhpEnvironment\Request $request)
+    {
        $this->request = $request->setPathInfo();
+       $this->objectManager = ObjectManager::getInstance();
```

Comment:
ObjectManager moet ook in `__construct(`. dan opnieuw je DI commando runnen

Reply:
Na overleg zijn wij er over uit gekomen dat dit niet nodig is.

Reply:
done